portality/app.py

```diff
@@ -279,6 +279,11 @@ def form_diff_table_subject_expand(val):
 
     return ", ".join(results)
 
+@app.template_filter("is_in_the_past")
+def is_in_the_past(dttm):
+    date = datetime.strptime(dttm, "%Y-%m-%d").date()
```

Comment:
datetime may need to be replaced with from portality.lib import dates as per https://github.com/DOAJ/doaj/pull/2208 but as it was implemented in another branch now, no sure if this PR should wait

Reply:
@richard-jones ?

Reply:
In any case should use the `dates` library if possible, even if it can't use the latest stuff that Philip wrote.  You could replace this with

```
dates.parse(dttm).date()
```

This will guess the date format correctly, but if you wanted it to be faster you can use

```
dates.parse(dttm, format="%Y-%m-%d").date()
```

Or you could add a utility function to the `dates` library, like

```
def is_before(mydate, comparison=None)
    if comparison is None:
        comparison = datetime.utcnow()
    if isinstance(mydate, str):
        mydate = parse(mydate)
    if isinstance(comparison, str):
        comparison = parse(comparison)
    return mydate < comparison
```